lib/poetry.js

```diff
@@ -21,12 +21,30 @@ async function pyprojectTomlToRequirements(modulePath, pluginInstance) {
     generateRequirementsProgress = progress.get(
       'python-generate-requirements-toml'
     );
-    generateRequirementsProgress.update(
-      'Generating requirements.txt from "pyproject.toml"'
+  }
+
+  const emitMsg = (msg) => {
+    if (generateRequirementsProgress) {
+      generateRequirementsProgress.update(msg)
+      log.info(msg);
+    } else {
+      serverless.cli.log(msg);
+    }
+  }
+
+  try {
+    await spawn("test", ["-f", "poetry.lock"])
+    emitMsg('Generating requirements.txt from poetry.lock...');
+  } catch (e) {
+    if (options.requirePoetryLockFile) {
+      throw new Error(
+        "poetry.lock file not found - set requirePoetryLockFile to false to "
+        + "disable this error"
+      );
+    }
+    emitMsg(
+      'Generating poetry.lock and requirements.txt from pyproject.toml...'
```

Comment:
Let's not put the dots at the end - it can update an interactive spinner which already indicates that something is happening